### DISCUSSION TITLE
計算結果の可視化

### DISCUSSION THREAD

**Discussion by rag769** [roratorio-hub/ratorio#723] [Ideas]
装備検討の動画で、複数の装備セットに対して複数のスキルダメージを比較するシーンを見ました。
Clipでは特定の装備、スキルに対して火力グラフをおまけ表示していますが、
この入口を拡張して、複数のスキルで統合的に比較できるようにするのはどうでしょう？

※作り手は興味が湧いたものを作るだけで良いですが、運用される側から見ると
※機能を増やすと内部データの持ち方を変える時とか、DOM構成を変える時に足を引っ張りかねず、
※維持・メンテコストもかかることなので、ニーズとの兼ね合いで…
※※画像保存とか「試す」分には楽しかったけど「常用」しない (ニーズの低い) 機能の追加は運用ベースだと判断し辛いですよね

## サンプルソース
[お試しで改造した js の zip (添付できなかったので)](https://github.com/user-attachments/files/18055958/calchistory.zip)

calcx.html に下記の追加が必要です。
> <script src="https://cdn.jsdelivr.net/npm/gasparesganga-jquery-loading-overlay@2.1.7/dist/loadingoverlay.min.js"></script>

## デモ
https://rag769.github.io/ratorio/ro4/m/calcx.html

### 簡易説明
![image](https://github.com/user-attachments/assets/d2b904c7-a9fa-4ee6-97e4-890721488b59)

**Comment by usachoco** [comment]
ありがとうございます！
立て込んでしまって手を動かせそうにないのでひとまずコメントだけ

初見だと戸惑う人もいるかも知れないので、既に実装済みのクリップ機能と合わせて説明動画込みでリリースしたい感じがしますね
最近、動画作りが少し楽しいのでやってみようかなって感じです

頂いたコードはまだ見れてないんですが、いつもシンプルにまとめてくださるので今回もきっと大丈夫と思ってます
